server/run.sh

```diff
@@ -53,6 +53,15 @@ _ "Put build,test, delivery scripts in proper place"
 #echo "ADD /test_script /usr/bin/" >> $TARGET_FILE
 #echo "ADD /delivery_script /usr/bin/" >> $TARGET_FILE
 
+# Alter target file, if openshift container is detected
+CONTAINER_NAME="${APPID}/${JOBID}";
+if [[ ${CONTAINER_NAME} =~ ^openshift\/(origin.*|node|openvswitch) ]]; then
+    _ "Openshift container detected, checking if  ${TARGET_FILE} needs to be altered";
+    if [[ ( ${DESIRED_TAG} == "v1.5" && ${CONTAINER_NAME} != "openshift/origin-base" ) && ${CONTAINER_NAME} != "openshift/origin-source" ]]; then
```

Comment:
why are we hardcoding to 1.5 here. cant we have any <desired_tag>. and exception for `latest` only?